vital/tests/test_detected_object.cxx

```diff
@@ -104,3 +104,18 @@ TEST(detected_object, modification)
 
   doby.set_bounding_box( bb );
 }
+
+// ----------------------------------------------------------------------------
+TEST(detected_object, keypoints)
+{
+  kwiver::vital::bounding_box_d bb( 50, 50, 100, 100 );
+  kwiver::vital::detected_object dobj( bb );
+
+  dobj.add_keypoint( "head", kwiver::vital::point_2d( { 11.2, 13.1 } ) );
+  dobj.add_keypoint( "head", kwiver::vital::point_2d( { 4.2, 9.5 } ) );
+
+  EXPECT_EQ( dobj.keypoints().size(), 1 );
```

Comment:
This should probably be an `ASSERT_EQ`. Also, I feel like you are missing an assert that the keypoints contains one named "head" prior to trying to access the same.